.github/workflows/CI.yml

```
@@ -0,0 +1,37 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: create application.yml
+        run: |
+          mkdir ./src/main/resources
+          cd ./src/main/resources
+          
+          touch ./application.yml
+          echo "${{ secrets.APPLICATION_YML }}" >> ./application.yml
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build with Gradle Wrapper
+        run: ./gradlew build
```

p2;

여기 ./gradlew -x test build로 바꾸면 test에서 통과하지 못하여 빌드가 되지 못하는 오류를 해결할 수 있을 거 같습니다
```suggestion
        run: ./gradlew -x test build
```